x-pack/plugins/stack_connectors/server/connector_types/lib/servicenow/schema.ts

```diff
@@ -58,8 +61,26 @@ const CommonAttributes = {
   subcategory: schema.nullable(schema.string()),
   correlation_id: schema.nullable(schema.string({ defaultValue: DEFAULT_ALERTS_GROUPING_KEY })),
   correlation_display: schema.nullable(schema.string()),
+  additional_fields: schema.nullable(
+    schema.recordOf(
+      schema.string({
+        validate: (value) => validateOtherFieldsKeys(value),
```

Comment:
nitpick: `validate: validateOtherFieldsKeys`

Reply:
We have discovered that if we do this the `this` binds to the `schema` class and not to the function. This leads to errors like `this.validateOtherFieldsKeys ` is `undefined` etc. By doing that we keep the binding to the correct function.